DICOMPlugins/DICOMTID1500Plugin.py

```diff
@@ -54,27 +61,38 @@ def examineFiles(self, files):
       isDicomTID1500 = self.isDICOMTID1500(dataset)
 
       if isDicomTID1500:
-        loadable = self.createLoadableAndAddReferences([dataset])
+        # This function now takes as input the dataset and sr reading from highdicom. 
+        sr = hd.sr.srread(cFile)
+        loadable = self.createLoadableAndAddReferences([dataset], sr)
```

Comment:
Why not keep the existing API and instead load it with highdicom inside `createLoadableAndAddReferences`?

Reply:
Now I keep the existing API, and used `from_dataset` to load the SR. 

Fixed in this [branch](https://github.com/deepakri201/QuantitativeReporting/tree/add_planar_annot).